integration-tests/smoke/vrfv3_test.go

```diff
@@ -0,0 +1,336 @@
+package smoke
+
+import (
+	"fmt"
+	"github.com/smartcontractkit/chainlink-env/environment"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/chainlink"
+	eth "github.com/smartcontractkit/chainlink-env/pkg/helm/ethereum"
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink/core/services/keystore/chaintype"
+	"github.com/smartcontractkit/chainlink/core/utils"
+	networks "github.com/smartcontractkit/chainlink/integration-tests"
+	"github.com/smartcontractkit/chainlink/integration-tests/actions"
+	"github.com/smartcontractkit/chainlink/integration-tests/client"
+	"github.com/smartcontractkit/chainlink/integration-tests/contracts"
+	ocr2vrftypes "github.com/smartcontractkit/ocr2vrf/types"
+	"github.com/stretchr/testify/require"
+	"math/big"
+	"strings"
+	"testing"
+
+	"github.com/rs/zerolog/log"
+)
+
+func TestVRFv3Basic(t *testing.T) {
+	linkEthFeedResponse := big.NewInt(1e18)
+	t.Parallel()
+	testEnvironment, testNetwork := setupVRFv3Test(t)
+
+	chainClient, err := blockchain.NewEVMClient(testNetwork, testEnvironment)
+	require.NoError(t, err)
+	contractDeployer, err := contracts.NewContractDeployer(chainClient)
+	require.NoError(t, err)
+	chainlinkNodes, err := client.ConnectChainlinkNodes(testEnvironment)
+	require.NoError(t, err)
+	nodeAddresses, err := actions.ChainlinkNodeAddresses(chainlinkNodes)
+	require.NoError(t, err, "Retreiving on-chain wallet addresses for chainlink nodes shouldn't fail")
+	//t.Cleanup(func() {
+	//	err := actions.TeardownSuite(t, testEnvironment, ctfutils.ProjectRoot, chainlinkNodes, nil, chainClient)
+	//	require.NoError(t, err, "Error tearing down environment")
+	//})
```

Comment:
should this be uncommented?

Reply:
yeah, I thought I added //TODO comment to uncomment it when im done with experiments

Reply:
done